packages/obojobo-chunks-multiple-choice-assessment/viewer-component.js

```diff
@@ -225,7 +225,7 @@ export default class MCAssessment extends React.Component {
 		)
 	}
 
-	componentWillReceiveProps() {
+	UNSAFE_componentWillReceiveProps() {
```

Comment:
Could this be replaced with componentDidUpdate?  That's the recommended React Method